docs/sources/reference/commandline/cli.md

```diff
@@ -533,6 +533,10 @@ considered as normal files. Filepaths in .dockerignore are absolute with
 the current directory as the root. Wildcards are allowed but the search
 is not recursive.
 
+When the `URL` references a Dockerfile, or a Dockerfile is read from 
+`STDIN`, then the `-f` option will have no effect and the contents of the
+Dockerfile will be placed in a file called `Dockerfile`.
+
```

Comment:
@moxiegirl so sorry.  For some reason I thought those edits were against the API docs, not the cil.md - I'll work on adding them.